<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModbusRtuWidget</class>
 <widget class="QWidget" name="ModbusRtuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>86</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,2,1,2,1,2,7" columnminimumwidth="1,2,1,2,1,2,7">
    <item row="1" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>校验位：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>波特率：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QComboBox" name="comboBox_DataBits"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>串口号：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>数据位：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_SerialName"/>
    </item>
    <item row="0" column="3">
     <widget class="QComboBox" name="comboBox_BaudRate"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_StopBits"/>
    </item>
    <item row="1" column="3">
     <widget class="QComboBox" name="comboBox_Parity"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>停止位：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QComboBox" name="comboBox_FlowCtrl"/>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>流控制：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6" rowspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
